src/sentry/static/sentry/app/components/broadcastModal.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import IconCloseLg from '../icons/icon-close-lg';
+import ConfigStore from '../stores/configStore.jsx';
```

Comment:
do you actually want/need the `.jsx`? i think it works both ways, but seems cleaner to not include